libres/lib/res_util/es_testdata.cpp

```diff
@@ -201,8 +201,8 @@ es_testdata::es_testdata(const char *path)
     this->R = alloc_load("R", this->active_obs_size, this->active_obs_size);
     this->D = alloc_load("D", this->active_obs_size, this->active_ens_size);
     this->dObs = alloc_load("dObs", this->active_obs_size, 2);
-    this->obs_mask = bool_vector_alloc(this->active_obs_size, true);
-    this->ens_mask = bool_vector_alloc(this->active_ens_size, true);
+    this->obs_mask = std::vector<bool>(this->active_obs_size, true);
+    this->ens_mask = std::vector<bool>(this->active_ens_size, true);
```

Comment:
You should not have to repeat the type since it can be inferred from the left hand side. _[Caveat: it could be that the first arg's type should be turned into `size_t`]_

```suggestion
    this->obs_mask = {this->active_obs_size, true};
    this->ens_mask = {this->active_ens_size, true};
```